core/modules/user/css/user.css

```diff
@@ -45,10 +45,14 @@
   max-width: 100%;
 }
 .password-toggle {
-  right: 0;
   position: absolute;
-  font-size: 80%;
   top: -2em;
+  right: 0; /* LTR */
+  font-size: 80%;
+}
+[dir="rtl"] .password-toggle {
+  right: unset;
```

Comment:
`unset` is relatively new (CSS 3) and doesn't have any IE support (even IE11). For the time being, could we just switch to `auto`, which is the default: https://www.w3schools.com/cssref/pr_pos_left.asp

Reply:
Agreed 👍 ...fixed 😉